app/_landing_pages/deck_landing.yaml

```diff
@@ -0,0 +1,225 @@
+
+                If you are using Kong alongside a database, you need decK because:
+
+                * Kong’s kong config db_import command is used to initialize a Kong database, but it is not recommended to use it if there are existing Kong nodes that are running, as the cache in these nodes will not be invalidated when entities are changed/added. You will need to manually restart all existing Kong nodes. decK performs all the changes via Kong’s Admin API, meaning the changes are always propagated to all nodes.
+                * Kong’s kong config db_import can only add and update entities in the database. It will not remove the entities that are present in the database but are not present in the configuration file.
+                * Kong’s kong config db_import command needs direct access to Kong’s database, which might or might not be possible in your production networking environment.
+                * decK can easily perform detect drifts in configuration. For example, it can verify if the configuration stored inside Kong’s database and that inside the config file is the same. This feature is designed in decK to integrate decK with a CI system or a cronjob which periodically checks for drifts and alerts a team if needed.
+                * deck dump or deck gateway dump outputs a more human-readable configuration file compared to Kong’s db_import.
+                However, decK has the following limitations which might or might not affect your use case:
+
+                If you have a very large installation, it can take some time for decK to sync up the configuration to Kong. This can be mitigated by adopting distributed configuration for your Kong installation and tweaking the --parallelism value. Kong’s db_import will usually be faster by orders of magnitude.
+                decK cannot export and re-import fields that are hashed in the database. This means fields like password of basic-auth credential cannot be correctly re-imported by decK. This happens because Kong’s Admin API call to sync the configuration will re-hash the already hashed password.
+    - column:
+      - type: structured_text
+        config:
+          blocks:
+            - type: text
+              text: "**I’m a Kong Gateway customer, can I use decK?**"
+            - type: text
+              text: Of course, decK is designed to be compatible with open-source and enterprise versions of Kong.
+  - row:
+    - column:
+      - type: structured_text
+        config:
+          blocks:
+            - type: text
+              text: "**Why the name ‘decK’?**"
+            - type: text
+              text: |
+                It is simple, short, and easy to use in the terminal. It is derived from the combination of words ‘declarative’ and ‘Kong’.
+    - column:
+      - type: structured_text
+        config:
+          blocks:
+            - type: text
+              text: "**Is there a JSON Schema for decK?**"
```

Comment:
Is this a tutorial 🤔 As in:

Q: Is there a JSON schema for decK? 
A: Yes, The decK team maintains a JSON schema that you can use to validate YAML files on Github. Learn how to use the schema with VScode >

Tutorial at link:

```
The decK team maintains a JSON schema that you can use to validate YAML files on Github. You can use the schema with a text editor to provide JIT YAML validation. For example, to use the JSON schema with VS Code:
               1. Install the Red Hat YAML extension:
                
                      code --install-extension redhat.vscode-yaml
               2. Edit the plugin settings in VS Code:
                       
                      "yaml.schemas": {
                        "https://json.schemastore.org/kong_json_schema.json": [
                            "kong.yml",
                            "kong.yaml"
                        ]
                    }
               3. Verify that it works by opening your existing `kong.yml` or `kong.yaml` file.
 ```